test/integration/newapp_test.go

```diff
@@ -2243,20 +2252,20 @@ func fakeImageStreamSearcher() app.Searcher {
 	})
 
 	return app.ImageStreamSearcher{
-		Client:            client.Image(),
-		ImageStreamImages: client.Image(),
+		Client:            client,
```

Comment:
If two identical clients are needed, then `ImageStreamSearcher` is too broad, try merging it to a single client.

Reply:
Not sure what you mean, do you want to change the fields in ImageStreamSearcher to have one field with a broader client? or change the integration test here to use something other than an app.ImageStreamSearcher?

Reply:
I've gone ahead and updated the ImageStreamSearcher to only have a single Client field that receives a broader interface